WBooks/Book Table/RentalsViewModel.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  RentalsViewModel.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 07/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveSwift
+
+class RentalsViewModel: BookTableAbstractViewModel {
+    
+    private var rents = [Rent]()
+    
+    let rentRepository = RepositoryBuilder.getDefaultRentsRepository()
+    
+    override func loadBooks() {
+        rentRepository.fetchRents().startWithResult { [weak self] result in
+            guard let this = self else {
+                return
+            }
+            switch result {
+            case .success(let resultArray):
+                this.rents = resultArray
```

Comment:
If you are not doing anything else with this array, you could use only books, and set it as `resultArray.map { ... }

Reply:
Right, my bad.